lib/multipart.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { readFileSync } = require('fs')
+const FormData = require('form-data')
+
+function getFormData (string) {
+  try {
+    return JSON.parse(string)
+  } catch (error) {
+    try {
+      return require(string)
```

Comment:
Can you please use fe.readFileSync instead? require is relative to the folder of autocannon, not the current directory.

Reply:
Yes. This was my lazy approach to load both `json` and `js` files and I didn't think of that.

Reply:
would something like this also solve the issue?
```js
const path = require.resolve(string, { paths: [process.cwd()] })
return require(path)
```

Reply:
I would prefer the use of fs.readFile anyway, it shows intent.

Reply:
Ok